physics/oblate_body_body.hpp

```diff
@@ -82,6 +84,19 @@ OblateBody<Frame>::Parameters::ReadFromMessage(
                           LegendreNormalizationFactor(2, 0) * reference_radius *
                           reference_radius;
 
+  // Zonalness.
+  parameters.is_zonal_ = true;
+  if (!(message.has_zonal() && message.zonal())) {
```

Comment:
```C++
if (!message.zonal()))
```
given that the default value is `false`?

Reply:
Done.